python/semantic_kernel/core_skills/time_skill.py

```diff
@@ -174,6 +176,48 @@ def minute(self) -> str:
         now = datetime.datetime.now()
         return now.strftime("%M")
 
+    @sk_function(
+        description="Get the date of offset from today by a provided number of days"
+    )
+    def days_ago(self, days: str) -> str:
+        """
+        Get the date a provided number of days in the past
+
+        params:
+            days: The number of days to offset from today
+        returns:
+            The date of the offset day.
+
+        Example:
+             SKContext["input"] = "3"
+             {{time.days_ago $input}} => Sunday, 7 May, 2023
+        """
+        d = datetime.date.today() - datetime.timedelta(days=int(days))
+        return d.strftime("%A, %d %B, %Y")
+
+    @sk_function(
+        description="Get the date of the last day matching the supplied day name in English"
+    )
+    def last_matching_day(self, day_name: str) -> str:
```

Comment:
is this function English only? if so, what's the plan to support localization?

Reply:
Sadly no plan as yet (I am not a python developer by day). Happy for any suggestions though. For now I have just ensured that the semantic description calls out the need for English in hope that the Planner (if used) can handle translation as needed

Reply:
@dluc Does this address your question (enough to complete this PR), or can you provide an example of what you have in mind for localization support?

Reply:
@tim-gill consdiering it's English only and that the description for AI could lead to some incorrect plans when using the Planner, I would ask to remove this function from the PR. If you could think about a description that would work well with the planner, otherwise I would suggest having this as a private skill in local projects

Reply:
@dluc - while I am happy enough to do this, I do want to point out that all of the other python functions around month and day in this file (not my changes) are also locked to English only as they also use the strftime function to generate their output - they also return different formats for the same data ("Sunday, 12 January" and "Sunday, January 12"). Do we also want to remove all of them?